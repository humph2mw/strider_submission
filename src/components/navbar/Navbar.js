import React from 'react';
import Navbar from 'react-bootstrap/Navbar';
import './style.css'

export default function Nav() {
  return (
    <div>
      <Navbar id="navbar">
        <Navbar.Brand>Strider General Store</Navbar.Brand>
      </Navbar>
    </div>
  )
}
