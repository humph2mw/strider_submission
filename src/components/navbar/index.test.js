import React from 'react'
import TestRenderer from 'react-test-renderer'
import Navbar from './index'

it('renders correctly', () => {
  const tree = TestRenderer
    .create(<Navbar />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
