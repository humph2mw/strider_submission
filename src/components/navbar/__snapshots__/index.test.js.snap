// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <nav
    className="navbar navbar-expand navbar-light"
    id="navbar"
  >
    <span
      className="navbar-brand"
    >
      <img
        id="striderLogo"
        src="/Strider-Logo.png"
      />
    </span>
  </nav>
</div>
`;
